README.md

```diff
@@ -215,17 +201,35 @@ Run the web app container using:
 
 # Troubleshooting
 
-<!-- TODO: Add Common Troubleshooting Issues -->
+There is a large amount of user uploaded images in a long running web app
+
+> When running the web app at `http://localhost:8088` an admin page is available at
+> [`http://localhost:8088/cleanup`](http://localhost:8088/cleanup) that allows the user to delete all user uploaded
+> files from the server.
+>
+> [Note: This deletes **all** user uploaded images]
 
-<!-- Idea: Explain cleanup page here? -->
+![Admin UI Screenshot](doc/source/images/cleanup.png)
 
 # Links
 
-<!-- TODO: Add Links -->
+* [Model Asset eXchange (MAX)](https://developer.ibm.com/code/exchanges/models/)
+* [Center for Open-Source Data & AI Technologies (CODAIT)](https://developer.ibm.com/code/open/centers/codait/)
+* [MAX Announcement Blog](https://developer.ibm.com/code/2018/03/20/igniting-a-community-around-deep-learning-models-with-model-asset-exchange-max/)
+
+## Libraries used in this Code Pattern
+* [D3.js](https://d3js.org): D3.js is a JavaScript library for manipulating documents based on data.
+* [d3-cloud](https://github.com/jasondavies/d3-cloud): A Wordle-inspired word cloud layout written in JavaScript.
+* [Featherlight](https://noelboss.github.io/featherlight/): Featherlight is a very lightweight jQuery lightbox plugin.
+* [Glyphicons](http://glyphicons.com): GLYPHICONS is a library of precisely prepared monochromatic icons and symbols, created with an emphasis to simplicity and easy orientation.
+* [Image Picker](http://rvera.github.io/image-picker/): Image Picker is a simple jQuery plugin that transforms a select element into a more user friendly graphical interface.
 
 # Learn More
 
-<!-- TODO: Add Learn More -->
+* **Data Analytics Code Patterns**: Enjoyed this Code Pattern? Check out our other [Data Analytics Code Patterns](https://developer.ibm.com/code/technologies/data-science/)
+* **AI and Data Code Pattern Playlist**: Bookmark our [playlist](https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewObx5N7uGZ5FKH0Fde) with all of our Code Pattern videos
+* **Watson Studio**: Master the art of data science with IBM's [Watson Studio](https://dataplatform.ibm.com/)
+* **Spark on IBM Cloud**: Need a Spark cluster? Create up to 30 Spark executors on IBM Cloud with our [Spark service](https://console.bluemix.net/catalog/services/apache-spark)
```

Comment:
@stevemart did you add these as examples? Because I'm not sure the Spark or Watson Studio link are very relevant to the subject of this web app.

Reply:
They're what we've been putting in for other CODAIT patterns since they've been mostly about Spark. If you don't like em, change em :)

They should be Call To Actions to look at other IBM tech.